.github/workflows/node.js.yml

```diff
@@ -5,16 +5,16 @@ name: Node.js CI
 
 on:
   push:
-    branches: [ master, v9, v10 ]
+    branches: [ master, v10, v11, prepare-v12 ]
   pull_request:
-    branches: [ master, v9, v10 ]
+    branches: [ master, v10, v11, prepare-v12 ]
```

Comment:
- [ ] Before merging, create `v11` branch and remove `prepare-v12` condition here